Classes/Tca/Registry.php

```diff
@@ -270,10 +274,19 @@ public function getPageTsString(): string
 ' . $cType . ' = ' . $containerConfiguration['backendTemplate'] . '
 }
 ';
-        }
-        $typo3Version = GeneralUtility::makeInstance(Typo3Version::class);
-        if ($typo3Version->getMajorVersion() > 12) {
             // s. https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/13.0/Breaking-102834-RemoveItemsFromNewContentElementWizard.html
+            if ($typo3Version->getMajorVersion() > 12) {
+                if ($containerConfiguration['registerInNewContentElementWizard'] === false) {
+                    $group = $containerConfiguration['group'] !== '' ? $containerConfiguration['group'] : $defaultGroup;
+                    $cTypesByGroup[$group][] = $cType;
+                }
+            }
+        }
+
+        if ($typo3Version->getMajorVersion() > 12 && !empty($cTypesByGroup)) {
+            foreach ($cTypesByGroup as $group => $ctypes) {
+                $pageTs .= LF . 'mod.wizards.newContentElement.wizardItems.' . $group . '.removeItems := addToList(' . implode(',', $ctypes) . ')';
```

Comment:
we have to increase the count of the ignored errors in each Build/phpstan-baseline-1<1-3>.neon to make CI happy
e.g.
```
+++ b/Build/phpstan-baseline-13.neon
@@ -3,7 +3,7 @@ parameters:
 
                -
                        message: "#^Constant LF not found\\.$#"
-                       count: 4
+                       count: 5
                        path: ../Classes/Tca/Registry.php
```